specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingProfile.json

```diff
@@ -69,327 +82,540 @@
+          "description": "The properties of a(n) BillingProfile",
+          "allOf": [
+            {
+              "$ref": "#/definitions/BillingProfileProperties"
+            }
+          ],
           "x-ms-client-flatten": true
         }
       },
-      "type": "object"
-    },
-    "BillingProfileCreationRequest": {
-      "description": "The request parameters for creating a new billing profile.",
-      "properties": {
-        "billTo": {
-          "$ref": "./types.json#/definitions/AddressDetails",
-          "description": "The address of the individual or organization that is responsible for the billing profile."
-        },
-        "displayName": {
-          "description": "The name of the billing profile.",
-          "type": "string"
-        },
-        "enabledAzurePlans": {
-          "description": "Enabled azure plans for the billing profile.",
-          "items": {
-            "$ref": "./types.json#/definitions/AzurePlan"
-          },
-          "type": "array"
-        },
-        "invoiceEmailOptIn": {
-          "description": "Flag controlling whether the invoices for the billing profile are sent through email.",
-          "type": "boolean"
-        },
-        "poNumber": {
-          "description": "The purchase order name that will appear on the invoices generated for the billing profile.",
-          "type": "string"
-        }
-      }
+      "x-ms-azure-resource": true
     },
     "BillingProfileListResult": {
-      "description": "The list of billing profiles.",
+      "description": "A container for a list of resources",
+      "type": "object",
       "properties": {
         "nextLink": {
           "description": "The link (url) to the next page of results.",
-          "readOnly": true,
-          "type": "string"
-        },
-        "totalCount": {
-          "description": "Total number of records.",
-          "format": "int32",
-          "readOnly": true,
-          "type": "number"
+          "type": "string",
+          "readOnly": true
         },
         "value": {
-          "description": "The list of billing profiles.",
+          "description": "The list of resources.",
+          "type": "array",
           "items": {
             "$ref": "#/definitions/BillingProfile"
           },
           "readOnly": true,
-          "type": "array"
+          "x-ms-identifiers": [
+            "id"
+          ]
         }
       }
     },
     "BillingProfileProperties": {
-      "description": "The properties of the billing profile.",
+      "description": "A billing profile.",
+      "type": "object",
       "properties": {
         "billingRelationshipType": {
           "description": "Identifies which services and purchases are paid by a billing profile.",
           "enum": [
+            "Unknown",
             "Direct",
             "IndirectCustomer",
             "IndirectPartner",
-            "CSPPartner"
+            "CSPPartner",
```

Comment:
Can we avoid making this program-specific (e.g. CSPPartner)? We don't call it FedGovIndirectCustomer either... Can't we just describe this as IndirectCustomer, IndirectPartner, DirectPartner, DirectCustomer?

Reply:
@ramkd12 @amberbhargava to comment here